### PR TITLE
Translation to Polish language

### PR DESCRIPTION
**Pull request by kinjelom**
The login and email modules.

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+doContinue=Kontynuuj
+doIgnore=Ignoruj
+doAccept=Akceptuj
+doDecline=Odrzuć
+doForgotPassword=Nie pamiętasz hasła?
+doClickHere=Kliknij tutaj
+doImpersonate=Wciel się
+kerberosNotConfigured=Kerberos nie jest skonfigurowany
+kerberosNotConfiguredTitle=Kerberos nie jest skonfigurowany
+bypassKerberosDetail=Albo nie jesteś zalogowany przez Kerberos albo twoja przeglądarka nie jest skonfigurowana do logowania Kerberos. Kliknij kontynuuj by zalogować się w inny sposób
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: Missing a dot at the end.

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+registerTitle=Rejestracja
+loginTitle=Zaloguj się do {0}
+loginTitleHtml={0}
+impersonateTitle=Wcielenie {0}
+impersonateTitleHtml=Wcielenie <strong>{0}</strong>
+realmChoice=Królestwo
+unknownUser=Nieznany użytkownik
+loginTotpTitle=Konfiguracja dla Mobile Authenticator
+loginProfileTitle=Zaktualizuj informacje konta
+loginTimeout=Zbyt dłużo czasu zajeło logowanie. Proces logowania rozpocnie się od nowa.
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: `rozpocnie` -> `rozpocznie`

Comment (slaskawi) [reply themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: `zajeło` -> `zajęło`

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+
+loginTotp.totp=Oparte o czas
+loginTotp.hotp=Oparte o licznik
+
+
+oauthGrantRequest=Czy przyznajesz te uprawnienia dostępu?
+inResource=w
+
+emailVerifyInstruction1=Została wysłana do Ciebie wiadomość e-mail z instrukcjami jak zweryfikować swój adres e-mail.
+emailVerifyInstruction2=Nie otrzymałem kod weryfikacyjny w wiadomości e-mail?
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: This sentence doesn't look right.. maybe `Nie otrzymałem kodu weryfikacyjnego w wiadomości e-mail?`

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+emailVerifyInstruction3=aby ponownie wysłać wiadomość e-mail.
+
+emailLinkIdpTitle=Link {0}
+emailLinkIdp1=Wiadomość e-mail z instrukcjami, aby powiązać konto {0} {1} z kontem {2} została wysłana do Ciebie.
+emailLinkIdp2=Nie otrzymałem kod weryfikacyjny w wiadomości e-mail?
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: The same here...

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+
+copyCodeInstruction=Proszę skopiować ten kod i wklej go do aplikacji\:
+
+pageExpiredTitle=Strona wygasła
+pageExpiredMsg1=Aby ponownie uruchomić proces logowania
+pageExpiredMsg2=Aby kontynuować proces logowania
+
+personalInfo=Informacje osobiste\:
+role_admin=Admin
+role_realm-admin=Królestwo Admin
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: This one is pretty tricky (and funny) at the same time. 

A term `realm` can be translated to `królestwo`, which is more like a `kingdom` in English :) Looking into the dictionary, none o the terms fits here to be honest. But if I had to choose, I would pick `strefa`, which can be translated into `zone`. Even though it's still wrong, it's slightly better than the `królestwo` I think :)

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+client_realm-management=Zarządzanie królestwem
+client_broker=Broker
+
+invalidUserMessage=Nieprawidłowa nazwa użytkownika lub hasło.
+invalidEmailMessage=Nieprawidłowy adres e-mail.
+accountDisabledMessage=Konto jest wyłączone, skontaktuj się z administratorem.
+accountTemporarilyDisabledMessage=Konto jest tymczasowo wyłączone, skontaktuj się z administratorem lub spróbuj ponownie później.
+expiredCodeMessage=Przekroczono limit czasu logowania. Proszę Zaloguj się ponownie.
+expiredActionMessage=Akcja wygasła. Proszę kontynuować logowanie.
+expiredActionTokenNoSessionMessage=Akcji wygasła.
```
Comment (slaskawi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: `Akcji` -> `Akcja`

**Review by slaskawi — CHANGES_REQUESTED** [review]

**Comment by slaskawi** [comment]
@wildloop Thanks a lot for the contribution and great work! I marked some minor things we could improve. Let me know once you have an updated Pull Request.

File: themes/src/main/resources-community/theme/base/email/messages/messages_pl.properties [hunk 7]
```diff
@@ -0,0 +1,56 @@
+# encoding: UTF-8
+emailVerificationSubject=Zweryfikuj email
+emailVerificationBody=Ktoś utworzył konto {2} z tym adresem e-mail. Jeśli to Ty, kliknij poniższy link, aby zweryfikować swój adres e-mail \n\n{0}\n\nLink ten wygaśnie w ciągu {3}.\n\nJeśli nie utworzyłeś tego konta, po prostu zignoruj tę wiadomość.
+emailVerificationBodyHtml=<p>Ktoś utworzył konto {2} z tym adresem e-mail. Jeśli to Ty, kliknij <a href="{0}">ten link</a> aby zweryfikować swój adres e-mail</p><p>Link ten wygaśnie w ciągu {3}</p><p>Jeśli nie utworzyłeś tego konta, po prostu zignoruj tę wiadomość.</p>
+emailTestSubject=[KEYCLOAK] - wiadomość testowa SMTP
+emailTestBody=To jest wiadomość testowa
+emailTestBodyHtml=<p>To jest wiadomość testowa</p>
+identityProviderLinkSubject=Link {0}
+identityProviderLinkBody=Ktoś chce połączyć Twoje konto "{1}" z kontem "{0}" użytkownika {2}. Jeśli to Ty, kliknij poniższy link by połączyć konta\n\n{3}\n\nTen link wygaśnie w ciągu {5}.\n\nJeśli nie chcesz połączyć konta to zignoruj tę wiadomość. Jeśli połączysz konta, będziesz mógł się zalogować na {1} przez {0}.
```
[the remark this answers is not in the document]
Comment (wildloop) [reply themes/src/main/resources-community/theme/base/email/messages/messages_pl.properties]: I think that a dot after link is useless, please take a look at this: https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/email/messages/messages_en.properties#L8

File: themes/src/main/resources-community/theme/base/email/messages/messages_pl.properties [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# encoding: UTF-8
+emailVerificationSubject=Zweryfikuj email
+emailVerificationBody=Ktoś utworzył konto {2} z tym adresem e-mail. Jeśli to Ty, kliknij poniższy link, aby zweryfikować swój adres e-mail \n\n{0}\n\nLink ten wygaśnie w ciągu {3}.\n\nJeśli nie utworzyłeś tego konta, po prostu zignoruj tę wiadomość.
```
[the remark this answers is not in the document]
Comment (wildloop) [reply themes/src/main/resources-community/theme/base/email/messages/messages_pl.properties]: done, thanks

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties (hunk 1 above)
Comment (wildloop) [reply themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: done, thanks

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties (hunk 2 above)
Comment (wildloop) [reply themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: done, thanks

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties (hunk 3 above)
Comment (wildloop) [reply themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: done, thanks

File: themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties (hunk 4 above)
Comment (wildloop) [reply themes/src/main/resources-community/theme/base/login/messages/messages_pl.properties]: done, thanks

**Comment by wildloop** [comment]
I've updated pull request, please check it now.

File: themes/src/main/resources-community/theme/base/email/messages/messages_pl.properties (hunk 7 above)
[the remark this answers is not in the document]
Comment (slaskawi) [reply themes/src/main/resources-community/theme/base/email/messages/messages_pl.properties]: oooh right! Thanks for pointing this out.

**Review by slaskawi — APPROVED** [review]
LGTM. The only thing we could do is to squash those commits.

**Comment by wildloop** [comment]
Sorry for late answer, I don't know how to squash it using github. Maybe I just make the new PR with only one commit?

**Comment by slaskawi** [comment]
This is a question to @stianst. When integrating a PR, there's a button "squash and merge" button but this feature needs to be enabled in the repo settings. 

@stianst This PR is good to me as is. Please squash it (if you can) and integrate it. Thanks!

**Comment by stianst** [comment]
@wildloop simply squash it in your local checkout, force push to wildloop:master, and the PR will be updated

@wildloop can you create a JIRA please?

**Comment by wildloop** [comment]
JIRA added: KEYCLOAK-8488

**Comment by wildloop** [comment]
Sorry, I have not time to fight merge-conflict while squashing, this is the new PR:
https://github.com/keycloak/keycloak/pull/5606

**Comment by stianst** [comment]
FIY If you'd just left the PR open and pushed to same branch the PR would have been updated.

**Pull request closed without merging**